### ISSUE TITLE
Windows: serial number of USB com port may be truncated

### ISSUE THREAD

**Issue by joergschoebel** [pyserial/pyserial#491]
Hi, 

when using serial.tools.list_ports_windows, the serial number of the USB serial port may be truncated. I have just come across this issue when using the default settings of an Atmel microcontroller for a usb-serial interface, such as in 
http://asf.atmel.com/docs/latest/uc3l/html/udc_use_case_2.html

The default serial number used by Atmel is "12...EF". Now, list_port_windows.py restricts the serial number of a hardware ID (see line 213 for "USB" devices, and probably also line 251 for "FTDIBUS" devices) in the grep command to \w , i.e. [a-zA-Z0-9_]. In my opinion, it should rather be \S (oder similar), to allow the full set of characters in the serial number. 

Kind Regards
Joerg

**Comment by vanmierlo** [comment]
Yes, this is very annoying. We have serial numbers with dashes ('-') and they are truncated as well.
Please fix this.

**Comment by bluehash** [comment]
I'm running into a similar issue. I have dashes '-' in the serial number, causing the Serial Number field to truncate.

Is there a reason why other characters are not included?
